python/ray/tests/test_job.py

```diff
@@ -93,8 +93,11 @@ def subtask():
         with open(v2_driver, "w") as f:
             f.write(driver_template.format(address, 2))
 
-        subprocess.check_call([sys.executable, v1_driver])
-        subprocess.check_call([sys.executable, v2_driver])
+        env = os.environ.copy()
+        # Make sure `lib` can be loaded
+        env.pop("PYTHONSAFEPATH", None)
```

Comment:
This is important since the `lib.py` file must be loaded from the local directory. In order to run tests "in place" without building ray, safe path isolation can be used, but then inside the test `lib` will not be found.